academic-services/knowledge-exploration-service/introduction.md

```diff
@@ -1,16 +1,88 @@
+
+## MAKES Architecture
+
+The MAKES API is designed to be hosted in an Azure Virtual Machine Scale Set to provide extensibility, scalability, and reliability.  Azure supports virtual machine scale sets with various features out of the box:
+
+- Auto scaling
+
+- Load balancing
+
+- Virtual Network for additional security
+
+Below is a diagram describing the internal MAKES architecture and how it interacts with requests:
+
+![Microsoft Academic Knowledge Exploration Service Architecture](media/makes-architecture.png "Microsoft Academic Knowledge Exploration Service Architecture")
+
+## Minimum Resource Requirements
+
+The most basic form of MAKES uses the following Azure assets:
+
+- *An Azure Storage account* - You will need to provision and specify a storage account when subscribing to MAKES.  Each release of MAKES will push all the required componenets to this storage account.  MAKES deployments require ~650GB of space.
+
+- *An Azure VM Image* - An Azure VM image will be provisioned by MAKES when deployed.
+
+- *Azure Virtual Machine Scale Set* - A scale set is provisioned by MAKES when deployed.  The scale set includes an Azure virtual network and security group, an Azure load balancer, a static public IP address and a premium SSD.
+
+![Required Azure Components](media/makes-components.png "Required Azure Components")
+
+## Scaling MAKES
+
+MAKES is designed to scale to our users' requirements.  If more throughput is required, users are able to deploy additional virtual machines using the Azure Virtual Machine Scale Set.  By default, a load balancer is placed in front of a VM Scale set to distribute the load.
+
+Additionally, if your application needs to be scaled globally, you may deploy MAKES to different regions.  Using Azure Traffic Manager or Azure Front Door traffic will be directed to the closest instance to ensure optimal performance.
+
+Below is an example of a geo-scalable architecture for MAKES:
+
+![Microsoft Academic Knowledge Exploration Service Scalable Service Architecture](media/makes-scalable-architecture-reference.png "Microsoft Academic Knowledge Exploration Service Scalable Service Architecture")
+
+## Using MAG and MAKES to generate a custom solution
+
+Some customers may have requirements that go beyond hosting the supplied index.  Specific customizations of the MAKES indexes are supported and give the user a great deal of flexibility.  Below is a diagram showing the end to end process users would follow when combining MAG and MAKES subscriptions to create custom solutions with private data and specific entities from MAG:
+
+![Creating Custom solutions with MAG and MAKES](media/combining-mag-and-makes.png "Creating Custom solutions with MAG and MAKES")
+
+This scenario can be taken further as well.  Users are able to modify the grammar and schema of MAKES to support additional languages and data attributes.  These advanced topics are covered in the "how to" section.
+
+## Knowledge Exploration Service
```

Comment:
Reading this page over a couple times I think we should just drop this section entirely.

I think it's best to explain it in the context of "advanced" topics, i.e. when we're showing them how to modify schema and/or grammar.

Reply:
I agree, can we move the section once we have the advanced topics up?  Currently I'm pointing to this bookmark in a couple different places and can update them all once we find a home.